.travis.yml

```diff
@@ -35,6 +38,8 @@ matrix:
         - cargo test --manifest-path sev/Cargo.toml --verbose --features=openssl
 install:
   - rustup component add rustfmt
+  - rustup target add x86_64-unknown-linux-musl
 script:
+  - export PKG_CONFIG_ALLOW_CROSS=1
```

Comment:
Can you document in the `README.md` the steps required to set up the development environment? This probably needs to include which packages need to be installed, which targets to add and which environment variables to set.

I'm not particularly excited about having to set an environment variable. This means integrations like rls won't work out of the box. Is there any way we can do without this?

Reply:
Difficult on fedora without musl-libc. There is https://copr.fedorainfracloud.org/coprs/taocris/musl/

Reply:
ok, works even without that env variable.. removed